site2/docs/administration-load-balance.md

```diff
@@ -143,6 +143,9 @@ loadBalancerSheddingGracePeriodMinutes=30
 
 Pulsar supports the following types of shedding strategies. From Pulsar 2.10, the **default** shedding strategy is `ThresholdShedder`.
 
+> **Note**<br /> 
```

Comment:
do you want this layout? 
<img width="673" alt="image" src="https://user-images.githubusercontent.com/50226895/165664516-28731db6-f673-4027-93e3-456e579b0c66.png">

if so, you do not need to add <br />

if no and you want there is an empty line before the next paragrahy, you can try <br /><br />

Reply:
Yes, I want this layout. But it only works with "br" added this time. Without it, it looks like below:
<img width="832" alt="image" src="https://user-images.githubusercontent.com/60642177/165689647-67ea3285-3928-4a1a-9b96-b1484d18c38e.png">

Not sure what's going wrong. I thought something had been changed in the framework.

Reply:
may try another layout like:

```
>**Note**
>
>xxx
```

Reply:
This solution leaves an empty line in between, which is not necessary. 
<img width="829" alt="image" src="https://user-images.githubusercontent.com/60642177/165875532-268bf279-589f-4e3d-bae0-2a2e559249f7.png">